src/app/shared/file.service.spec.ts

```diff
@@ -48,4 +59,18 @@ describe('FileService', () => {
         '/api/ga4gh/v2/tools/%23workflow%2Fgithub.com%2FHumanCellAtlas%2Fskylab%2FHCA_SmartSeq2/versions/dockstore/PLAIN-WDL/descriptor/HISAT2.wdl'
     );
   }));
+
+  it('should detect http import in WDL', inject([FileService], (fileService: FileService) => {
```

Comment:
Think we could use some negative tests for something like this. 
For example, if people added hyperlinks in the middle of  CWL `doc:` fields or CWL+WDL `#` comments  (watch out for https://github.com/broadinstitute/cromwell/issues/2870),  and WDL descriptions 
https://docs.dockstore.org/docs/publisher-tutorials/wdl-best-practices/

Reply:
I'm not handling the multiline doc case and I don't think I easily can with a simple regex. I think I should go back to a warning message instead of disabling the button, just in case there is a false positive.

Reply:
I meant that a simple link (say to an image or to a lab page) in one of the above situations shouldn't trigger your regex, but you should add a test to verify. 

I don't think you need to worry about the case that someone literally writes `import http://foo.com` in a comment, multi-line or now if that's what you meant.